content/en/users/datatransfer/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Data Transfer"
+linkTitle: "Data Transfer"
+type: docs
+description: "Documentation related to EGI Data Transfer"
+weight: 30
+---
+
+## Overview 
+
+The EGI Data Transfer Service is based on the FTS3 service, developed at CERN. It allows you to move any type of data files asynchronously from one storage to another. The service includes dedicated interfaces to display statistics of on-going transfers and manage storage resources parameters.
+
+The EGI  Data Transfer is ideal to move large amounts of files or very large files as the service has mechanisms to ensure automatic retry in case of failures and checksumming.
```

Comment:
```suggestion
The EGI  Data Transfer is ideal to move large amounts of files or very large files as the service has mechanisms to verify checksums and ensure automatic retry in case of failures.
```